packages/botonic-cli/src/commands/train.ts

```diff
@@ -1,49 +1,100 @@
+
+  run(): void {
+    if (this.isTaskPluginInstalled()) {
+      spawnNpmScript(`train:${this.name}`, () => `Finished training `)
+    } else {
+      this.logTaskPluginNotInstalled()
+    }
+  }
+
+  private isTaskPluginInstalled(): boolean {
+    const path = join(
+      BOTONIC_PROJECT_PATH,
+      'node_modules',
+      BOTONIC_NPM_NAMESPACE,
+      `plugin-${this.name}`
+    )
+    return existsSync(path)
+  }
+
+  private logTaskPluginNotInstalled(): void {
+    console.error(
+      colors.red(
+        `Training process has been stopped because you don't have @botonic/plugin-${this.name} installed.\nPlease, install it with the following command:`
+      )
+    )
+    console.log(colors.bold(`$ npm install @botonic/plugin-${this.name}`))
+  }
+}
+
+class InvalidTaskError extends Error {
+  constructor(taskName: string, availableTasks: string[]) {
+    super(
+      `Unsupported task '${taskName}'. Available tasks: '${availableTasks.join(
+        "', '"
+      )}'.`
+    )
+  }
+}
+
+export class Tasks {
+  static tasks = {
+    ner: new Task('ner'),
+    'text-classification': new Task('text-classification'),
+  }
+
+  static getByName(taskName: string): Task {
+    if (!this.isValidTask(taskName)) {
+      throw new InvalidTaskError(taskName, Object.keys(this.tasks))
+    }
+    return this.tasks[taskName]
+  }
+
+  private static isValidTask(taskName: string): boolean {
+    return Object.keys(this.tasks).includes(taskName)
+  }
+}
+
 export default class Run extends Command {
-  static description = 'Train your bot with NLU'
+  static description = 'Train your bot with NLP'
 
   static examples = [
-    `$ botonic train
-    TRAINING MODEL FOR {LANGUAGE}...
+    `$ botonic train [--task=<${Object.keys(Tasks.tasks).join('|')}>]
+    TRAINING MODEL...
     `,
   ]
 
   static flags = {
-    lang: flags.string(),
+    task: flags.string(),
   }
 
   static args = []
 
   private telemetry = new Telemetry()
 
   async run(): Promise<void> {
-    this.telemetry.trackTrain()
-    const botonicNLUPath: string = path.join(
-      process.cwd(),
-      'node_modules',
-      '@botonic',
-      'nlu'
-    )
     try {
-      await import(botonicNLUPath)
+      this.telemetry.trackTrain()
+      const { flags } = this.parse(Run)
+      const tasks = this.getTasks(flags.task)
+      tasks.forEach(task => task.run())
     } catch (e) {
-      const error = String(e)
-      this.telemetry.trackError(`Running botonic train: ${error}`)
-      console.log(e)
-      console.log(
-        colors.red(
-          `You don't have @botonic/plugin-nlu installed.\nPlease, install it with the following command:`
-        )
-      )
-      console.log(`$ npm install @botonic/plugin-nlu`)
-      return
+      console.error(e)
     }
-    spawnNpmScript('train', () => 'Finished training')
+  }
+
+  private getTasks(taskName: string | undefined): Task[] {
+    return taskName ? [Tasks.getByName(taskName)] : Object.values(Tasks.tasks)
```

Comment:
move getTasks within Tasks.
A hint that it should be there is that it only gets data from Tasks class :-)
Tasks.tasks might can it be made private